README.md

```diff
@@ -1,28 +1,28 @@
 # TrackWeave - Ramifiable and Directed Transactions
 
-TrackWeave is the Ramifiable and Directed Transactions (RDT) data structure,
+TrackWeave is a Ramifiable and Directed Transactions (RDT) data structure,
```

Comment:
TrackWeave is like a trademark name for the data structure, there is no such thing as an RDT until I invented it lol.

Reply:
```suggestion
TrackWeave is the Ramifiable and Directed Transactions (RDT) data structure,
```